tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+FROM_TAG=$(grep "git clone --single-branch" Documentation/Getting-Started/quickstart.md | awk '{ print $6 }')
+TO_TAG="$1"
+
+# This script is supposed to facilitate the preparation of a rook release by
+# changing example and documentation files to use the new release version instead the old one.
+#
+# It will usually be run on a release branch and the resulting changes will be the content of the
+# commit for the basis of the release.
+
+# The script needs two version tags:
+
+# FROM_TAG: the version tag of the previous/current release (e. g. v1.16.7)
+# TO_TAG: the version tag of the new release being prepared (e. g. v1.16.8)
+
+# FROM_TAG is auto-detected and TO_TAG should be passed to the script as the first positional parameter in its invocation.
+
+# So an example invocation could be:
+# set-release-ver.sh v1.16.8
+#
+# This assumes invocation from the release-1.16 branch with latest
+# release tag v1.16.7
+#
+# The script replaces all occurrences of $FROM_TAG with $TO_TAG in
+# the relevant affected files.
+
+FILES_DOCS=(
+Documentation/Getting-Started/quickstart.md
+Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md
+Documentation/Upgrade/rook-upgrade.md
+)
+FILES_CHARTS=(
+deploy/charts/rook-ceph/values.yaml
+)
+FILES_EXAMPLES=(
+deploy/examples/direct-mount.yaml
+deploy/examples/images.txt
+deploy/examples/operator-openshift.yaml
+deploy/examples/operator.yaml
+deploy/examples/osd-purge.yaml
+deploy/examples/toolbox-job.yaml
+deploy/examples/toolbox-operator-image.yaml
+)
+
+sed -i -e "s/docker.io\/rook\/ceph:$FROM_TAG/docker.io\/rook\/ceph:$TO_TAG/g" "${FILES_EXAMPLES[@]}"
+sed -i -e  "s/tag: $FROM_TAG/tag: $TO_TAG/g" "${FILES_CHARTS[@]}"
+sed -i -e  "s/$FROM_TAG/$TO_TAG/g" "${FILES_DOCS[@]}"
```

Comment:
The upgrade guide will need some more special handling, to match what is done for example in #15737.